idaes/ui/flowsheet.py

```diff
@@ -449,6 +449,26 @@ def _construct_model_json(self):
             )
         )
 
+        if ('Units' in self._stream_table_df and
+            'Variable' in self._stream_table_df):
+            # Styling the units for display
+            def apply_html(value, style=''):
```

Comment:
Wouldn't this make more sense in the JS client? Seems wrong to hardcode display styles in the server.